copilot_proxy/.dockerignore

```diff
@@ -0,0 +1,322 @@
+.idea/**/workspace.xml
```

Comment:
These `.dockerignore` files look more like copying a template somewhere, so many contents are very redundant. I agree with ignoring the entire `.idea` folder completely, and since the entire `.idea` folder is [already ignored in `.gitignore`](https://github.com/fauxpilot/fauxpilot/blob/main/.gitignore#LL13C7-L13C7), maybe those files won't have a chance to show up in the repository either.